lib/taskana-core/src/main/java/pro/taskana/task/internal/models/TaskSummaryImpl.java

```diff
@@ -65,7 +66,9 @@
   protected String custom15;
   protected String custom16;
 
-  public TaskSummaryImpl() {}
+  public TaskSummaryImpl() {
+    manualPriority = -1;
+  }
```

Comment:
I prefer setting the default value directly where you declare `manualPriority` instead of doing it here in the constructor.
@gitgoodjhe what do you prefer? Across the project we've done it both ways.

Reply:
at least we could make it a static field to make the default more obvious. Regarding the initialization I agree but both ways are fine for me.